src/graphql/acsys/mod.rs

```diff
@@ -336,21 +336,21 @@ and this parameter will be removed."]
 // specification.
 
 fn strip_event(drf: &str) -> &str {
-    &drf[0..drf.find('@').unwrap_or(drf.len())].trim_end()
+    drf[0..drf.find('@').unwrap_or(drf.len())].trim_end()
```

Comment:
Nice catch! I'm not sure why I didn't see these (I haven't run 'clippy' in a while.)